modules/recurlyjs/src/Controller/RecurlyJsSubscriptionSignupController.php

```diff
@@ -29,7 +35,11 @@ class RecurlyJsSubscriptionSignupController extends ControllerBase {
    * @return array
    *   A Drupal render array.
    */
-  public function subscribe(EntityInterface $entity, $plan_code, $currency = NULL) {
+  public function subscribe(RouteMatchInterface $route_match) {
+    $entity_type_id = \Drupal::config('recurly.settings')->get('recurly_entity_type') ?: 'user';
```

Comment:
@adamzimmermann are you deliberately linking to issue or just pointing out the redundancy?